### ISSUE TITLE
Assyrian Highlands

### ISSUE THREAD

**Issue by RedstoneCobra** [simsulla/nwomod#1040]
Proposition to create:
- Assyrian Highlands province (Province name will be "Dahuk") (w/o Mosul) (Iraqi Kurdistan currently controls this area and plans to make it an autonomous region once independence is secured 
- Assyrian Culture
- Assyrian Core on province

https://upload.wikimedia.org/wikipedia/commons/1/1c/Assyria_World_War_1_relief_English.png
Area around Mosul will be left out

**Comment by RedstoneCobra** [comment]
Even in the 1946 scenario?

**Comment by simsulla** [comment]
One would need to do more research, Maybe there is some Iraq census figures to use from that time. But I would be surprised if there has been changes of such magnitude that Assyrians from a current numbering c1% would then be close to majority in the province.

**Comment by RedstoneCobra** [comment]
Also not to mention, in the 92 start there should still be considerable numbers of Assyrians. As this is prior to Saddam and the Islamic State

**Comment by simsulla** [comment]
The c1% gov elections 2008 are pre ISIS. Saddam to my knowledge was in comparison relatively secular.
Religious official Iraqi census figures would decide the issue.

**Comment by RedstoneCobra** [comment]
Just read, the last CIA information from 87' had their population at around 3-5% of the iraqi total population

**Comment by arandomperson5000** [comment]
Here is the oldest post Ottoman Iraqi census I could find it was taken in 1957. Looking at the demographics for the Mosul governorate it appears that Christians makes ip approximately one seventh of the entire population of the governorate.
https://en.wikipedia.org/wiki/Iraqi_Census_(1957)

**Comment by simsulla** [comment]
Less than [12%](https://en.wikipedia.org/wiki/Iraqi_Census_(1957)) in Mosul (the province with most Christians), I would say does not warrant a Assyrian core for that province. Suggest closing this issue,

**Comment by RedstoneCobra** [comment]
Did you read the start of the issue? My intent is to leave this province separate from Mosul.

**Comment by arandomperson5000** [comment]
I think there is a couple of flaws with the depiction of Christians in the Middle East. The first one I would have to say is about the Assyrians, as detailed in previous posts if one were to change the Christian Arab pops to Assyrian pops in Iraq they would have to create new ethnic groups such as the Copts on the basis of religion. However if you based on linguistics the Christian Arab pops of countries outside Iraq wouldn't have to be changed as there is still a sizable portion of Assyrians that still speak Assyrian a language that is completely separate from Arabic. Another thing I noticed is that a large portion of the pops represent into the Maronites in Lebanon are Orthodox when they're Catholic in real life just a minor thing I noticed.
